Ziffity/AddCustomer/Model/Import/JsonImport.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace Ziffity\AddCustomer\Model\Import;
+
+use Magento\Framework\Filesystem\Driver\File;
+use Magento\Framework\Serialize\SerializerInterface;
+use Magento\Framework\Exception\InvalidArgumentException;
+use Ziffity\AddCustomer\Model\Customer;
+use Ziffity\AddCustomer\Profiles\ProfileInterface;
+
+
+class JsonImport implements ProfileInterface
+{
+    /**
+     * @var SerializerInterface
+     */
+    private $serializer;
+    /**
+     * @var File
+     */
+    private $file;
+
+    /**
+     * @param Customer $customer
+     * @param File $file
+     * @param SerializerInterface $serializer
+     */
+    public function __construct(
+        File $file,
+        SerializerInterface $serializer
+    ) {
+        $this->serializer = $serializer;
+        $this->file = $file;
+    }
+
+    /**
+     * @param string $fixture
+     * @return mixed|void
+     * @throws InvalidArgumentException
+     * @throws \Magento\Framework\Exception\FileSystemException
```

Comment:
@msakila532 all cases use the alias instead of the full path. it will help for code readability.

Reply:
Issue resolved